feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -250,7 +266,7 @@ func TestBasic(t *testing.T) {
 		// Note: This is not a subtest because a failure here means checking the
 		//   rest of the counters is pointless - none of them will change if we
 		//   haven't been exchanging IS-IS messages.
-		deadline = time.Now().Add(time.Second * 5)
+		deadline = time.Now().Add(time.Second * 15)
```

Comment:
same thing, why 15 sec.

Reply:
it is calculated based on number of rpc calls, each rpc call needs 3 secs.